test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git"
```

Comment:
- [ ] Great job implementing this project. In the requirements provided to you, you ought to git ignore this file. Kindly implement this and you would be through for the day. 👍